lib/chef/resource/chef_client_systemd_timer.rb

```diff
@@ -48,43 +48,52 @@ class ChefClientSystemdTimer < Chef::Resource
       DOC
 
       property :job_name, String,
-        default: Chef::Dist::CLIENT,
-        description: "The name of the system timer to create."
-      property :description, String, default: "Chef Infra Client periodic execution"
+        description: "The name of the system timer to create.",
+        default: Chef::Dist::CLIENT
+
+      property :description, String,
+        description: "The description to add to the systemd timer. This will be displayed when running `systemctl status` for the timer.",
+        default: "Chef Infra Client periodic execution"
```

Comment:
Does this need to be 
```suggestion
        default: "#{Chef::Dist::PRODUCT} periodic execution"
```